RELEASE_NOTES.md

```diff
@@ -6,6 +6,8 @@ A new header is inserted each time a *tag* is created.
 ## v1.15.2 (currently main branch)
 
 - engine: add support for PCSS (Percentage Closer Soft Shadows)
+- engine: add experimental non-glossy screen-space reflections [⚠️ **Materials need to be rebuilt to
```

Comment:
You should sync, Mathias just revved materials for v1.16

Reply:
That PR hasn't been merged yet, but I'll update this after he does.